calico_containers/calico_ctl/node.py

```diff
@@ -399,11 +375,11 @@ def warn_if_unknown_ip(ip, ip6):
     :param ip6: IPv6 address which should be present on the host.
     :return: None
     """
-    if ip not in get_host_ips(4):
+    if ip not in get_host_ips(exclude=["docker0"]):
```

Comment:
These if statements should be symmetric.

Either:
1) They are both gated on the definition of ip/ip6, or
2) Neither are gated on the definition of ip/ip6, and the check is performed for both of them always.

If option 2, then you should clean up the if statements to be a bit more clear.

e.g.
```
if ip6 and (ip6 not in get_host_ips(version=6, exclude=["docker0"])):
  ....
```